packages/cli/src/templates/project-template/package.json

```diff
@@ -21,8 +21,8 @@
     "@shopware-pwa/nuxt-module": "latest",
     "@storefront-ui/vue": "0.13.2",
     "@vue/test-utils": "^2.0.0",
-    "@vuelidate/core": "^2.0.0-alpha.41",
-    "@vuelidate/validators": "^2.0.0-alpha.29",
+    "@vuelidate/core": "2.0.0",
```

Comment:
The newest one is `2.0.2` maybe it's worth doing this?

Reply:
No, 2.0.2 is the cause of this bug 
![Screen Shot 2023-03-31 at 10 24 03 AM](https://user-images.githubusercontent.com/29194019/229015111-0dadda8f-d31f-4da4-ae1d-f099aa7bb0fc.png)

Reply:
ok so let's leave that for now :)